core/src/tests/dummy_muxer.rs

```diff
@@ -0,0 +1,78 @@
+extern crate futures;
```

Comment:
This module is missing the copyright header and comments. "dummy_muxer" is too vague to indicate what this code does, so comments are needed.